executor/batch_point_get.go

```diff
@@ -379,6 +382,17 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	return nil
 }
 
+func (e *BatchPointGetExec) verifyTxnScope() error {
+	txnScope := e.txn.GetUnionStore().GetOption(kv.TxnScope).(string)
+	tblID := e.tblInfo.ID
+	is := infoschema.GetInfoSchema(e.ctx)
+	valid := placement.VerifyTxnScope(txnScope, tblID, is.RuleBundles())
```

Comment:
Does this `tblID` include partition IDs?

Reply:
Yes, the function sign is revised now.